### PR TITLE
Edits intro

### PR DESCRIPTION
**Pull request by lacmajedrez**
Corrected three typos in Intro.rst

Suggested reviewers:

- [Thibaut ] @thast

### CODE REVIEW

File: content/introduction/introduction_notation.rst [hunk 1]
```diff
@@ -4,20 +4,19 @@ Notation and Conventions
...
 We choose the notation set forth in :cite:`ward1988`. Their chapter has
-been the foundation of many research papers, is used by geophysicists world-
-wide, and it is clean and unambiguous.
+been the foundation of many research papers, is used by geophysicists world-wide, and it is clean and unambiguous.
 
 Vectors
 -------
 
-- vectors are bold:                          
-    * ie. :math:`\mathbf{v}`, :math:`\boldsymbol{\nabla\cdot}`
```
Comment (lacmajedrez) [inline content/introduction/introduction_notation.rst]: We are giving an example, thus it's better to use (e.g. = for example).

File: content/introduction/introduction_notation.rst [hunk 2]
```diff
@@ -26,7 +25,7 @@ Integrals
     .. math::
         \int_V f ~dv
 
-   or over a closed volume
+   or over a closed volume:
```
Comment (lacmajedrez) [inline content/introduction/introduction_notation.rst]: Correct me if I'm wrong, but I don't think such an integral exist, i.e., over a closed volume (what does that mean?)

File: content/introduction/introduction_notation.rst [hunk 3]
```diff
@@ -38,11 +37,11 @@ Integrals
     .. math::
         \oint_S \mathbf{f} \cdot \mathbf{da} = \oint_S \mathbf{f} \cdot \mathbf{\hat{n}} ~da
 
-- Integrating a vector function over a line: 
+- Integrating a vector function over a curve:
```
Comment (lacmajedrez) [inline content/introduction/introduction_notation.rst]: This kind of integrals are typically known as line integrals.  However, typically we integrate over curves.

File: content/introduction/introduction_notation.rst (hunk 2 above)
Comment (thast) [inline content/introduction/introduction_notation.rst]: Indeed the formulation seems weird. Feel free to edit it.

**Comment by thast** [comment]
So far I think it is good to go. Thanks @lacmajedrez

**Comment by lheagy** [comment]
Nice! Thanks @lacmajedrez  and @thast!

**Pull request merged**